_pieces_lib/pieces_os_client/wrapper/client.py

```diff
@@ -169,7 +170,7 @@ def ensure_initialization(self):
             Waits for all the assets/conversations and all the started websockets to open
         """
         self._check_startup()
-        BaseWebsocket.wait_all()
+        # BaseWebsocket.wait_all()
```

Comment:
did you mean to do this?

Reply:
yeah to avoid stoping the main thread causing too many issues just get want is loaded you don't have to wait for all assets to be loaded or all conversations to be loaded